arrows/qt/CMakeLists.txt

```diff
@@ -22,10 +28,6 @@ set( qt_sources
   image_io.cxx
   )
 
-if( KWIVER_ENABLE_PROCESSES )
```

Comment:
This (needing this conditional to appear twice) seems mildly unfortunate; how would you feel about moving the `kwiver_install_headers` calls to the end instead? (Not sure why they happen before the library rules anyway...)

Reply:
Yeah, that is what was messing me up
The install was way too early, before all files were properly identified

Installs at toward the end makes sense to me

Reply:
Ook, this is crisper